pytest_inmanta/plugin.py

```diff
@@ -619,13 +621,34 @@ def _load(self) -> None:
         module.Project.set(test_project)
         test_project.load()
 
-    def compile(self, main, export=False):
+    def compile(self, main: str, export: bool = False, no_dedent: bool = False):
```

Comment:
```suggestion
    def compile(self, main: str, export: bool = False, no_dedent: bool = False) -> None:
```

When you add typing to a method (which I really encourage) you should always do the return type as well. If there is one argument not typed (including the return type) a typechecker like mypy will mark it as incomplete

Reply:
Damn it, I always forget the None return type, thanks